src/routes/index.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import auth from './auth';
+
+const router = express.Router();
+
+router.use('/', auth);
```

Comment:
I think the index file here should refer to multiple sub-routes not just auth
In a situation where I need to add and set of routes for articles, I don't think it would be semantically right to use the auth file as an offshoot to those routes.
`const authUrl = '/auth';`
`router.use(${baseUrl}, auth)`
that would allow others to use the index file for their own routes too and in the auth file other auth routes could be added like sign up, sign in and the rest.

Reply:
I worked on a feature in the auth route. If someone works on the user route, they can simply import it in this index file and add it like this: `router.use('/', auth, user);`. If you understand this file properly, you would see that it's not using the auth file as an offshoot to the others.

Reply:
cool, I've never seen it done this way. looking at your reply, I see that the way you did it works just fine too and its a little more organized than the one I'm accustomed to. I've learned something new, Thanks.